daml-foundations/daml-tools/daml-extension/syntaxes/daml12.tmLanguage-update.xml

```diff
@@ -514,14 +511,22 @@
 			<string>#infix_op</string>
 		</dict>
 		<dict>
-			<key>begin</key>
+			<key>comment</key>
+			<string>We treat :: as cons. It's important this definition precede the rule for : (and that rule precede the one for arbitrary operators).</string>
+			<key>match</key>
 			<string>(::|∷)</string>
```

Comment:
I'm not sure we really want the unicode version of `::`. @neil-da @shayne-fletcher-da any thoughts?

Reply:
Yeah, let's ban Unicode other than in identifiers (and even then, I'd rather than it universally...). However, I think the grammar probably permits it, so might as well have the syntax match the grammar, and removing it from the grammar seems like the wrong call without persuading GHC upstream to drop silly unicode.

TLDR: I'd keep it.

Reply:
Its straight from Justus's work. Let's us not remove it here.

Reply:
FWIW, @neil-da recent changes in the GHC parser suggest that the community is in fact "doubling-down" on the unicode silliness :(